### PR TITLE
Reader: add basic placeholder for full post view

### LINKED ISSUE #9668: Loading placeholder for full post
Thus spake @blowery in the Slacks:

"we badly need a loading placeholder for full post"

### PR DESCRIPTION
**Pull request by bluefuton**
This PR adds a loading placeholder shown before the post is available in full post view.

It's pretty tricky to make a placeholder that fits all cases, because the content can be so different. I've used a pretty common scenario: post has a gravatar but no site icon, and a date but no tags.

Fixes #9668.

### CODE REVIEW

**Comment by bluefuton** [comment]
Todo: 
- [ ] add sidebar placeholders

File: client/blocks/author-compact-profile/placeholder.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+/**
+ * Internal dependencies
+ */
+import ReaderAvatar from 'blocks/reader-avatar';
+
+const AuthorCompactProfilePlaceholder = () => {
+	return (
+		<div className="author-compact-profile is-placeholder">
+			<div className="author-compact-profile__avatar-link">
+				<ReaderAvatar showBlankGravatar={ true } />
```
Comment (bluefuton) [inline client/blocks/author-compact-profile/placeholder.jsx]: This prop would probably be better named `showPlaceholder`.

File: client/blocks/reader-full-post/index.jsx [hunk 2]
```diff
@@ -270,6 +261,7 @@ export class FullPostView extends React.Component {
 		}
 
 		const externalHref = isDiscoverPost( referralPost ) ? referralPost.URL : post.URL;
+		const isLoading = true; //! post || post._state === 'pending';
```
Comment (bluefuton) [inline client/blocks/reader-full-post/index.jsx]: Need to uncomment this before review 👀

File: client/blocks/reader-full-post/index.jsx [hunk 3]
```diff
@@ -421,7 +412,7 @@ export default class FullPostFluxContainer extends React.Component {
 	}
 
 	static propTypes = {
-		blogId: React.PropTypes.string.isRequired,
+		blogId: React.PropTypes.string,
```
Comment (blowery) [inline client/blocks/reader-full-post/index.jsx]: yay

**Pull request merged**